web/src/test/java/org/springframework/security/web/util/matcher/MediaTypeRequestMatcherTests.java

```diff
@@ -53,8 +53,8 @@ public void setup() {
 	}
 
 	@Test(expected = IllegalArgumentException.class)
-	public void constructorNullCNSVarargs() {
-		new MediaTypeRequestMatcher(null, MediaType.ALL);
+	public void constructorWhenNullCNSThenIAE() {ContentNegotiationStrategy c = null;
```

Comment:
Will you please clean up the whitespacing here?

Reply:
ok